### PR TITLE
chore: login/logout basic implementation

### PR DESCRIPTION
**Pull request by dipeshkemuri**

### CODE REVIEW

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -2,79 +2,191 @@
...
+        <!--              <MenuItem v-slot="{ active }">-->
+        <!--                <a-->
+        <!--                  href="#"-->
+        <!--                  :class="[active ? 'bg-gray-100' : '', 'block px-4 py-2 text-sm text-gray-700']"-->
+        <!--                  @click="logoutHandler"-->
+        <!--                >Sign Out</a>-->
+        <!--              </MenuItem>-->
+        <!--            </MenuItems>-->
+        <!--          </transition>-->
+        <!--        </Menu>-->
```
Comment (akshay-kh) [inline src/components/Navbar.vue:63]: why do we ned to keep this section commented cant we just remove it its any way available in older version of git already do we have to keep this commented ?

File: src/components/Navbar.vue [hunk 2]
```diff
@@ -2,79 +2,191 @@
...
+          <!--            </div>-->
+          <!--            <div class="py-6 px-5">-->
+          <!--              <div class="grid grid-cols-1 gap-4">-->
+          <!--                <a href="#"-->
+          <!--                   class="text-base font-medium text-gray-900 hover:text-gray-700"-->
+          <!--                   @click="logoutHandler"-->
+          <!--                >Sign Out</a>-->
+          <!--              </div>-->
+          <!--            </div>-->
+          <!--          </div>-->
```
Comment (akshay-kh) [inline src/components/Navbar.vue:147]: this as well ?

File: src/pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        <!--                <span class="sr-only">Sign in with GitHub</span>-->
+        <!--                <svg class="w-5 h-5" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20">-->
+        <!--                  <path fill-rule="evenodd" d="M10 0C4.477 0 0 4.484 0 10.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0110 4.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.203 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.942.359.31.678.921.678 1.856 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0020 10.017C20 4.484 15.522 0 10 0z" clip-rule="evenodd" />-->
+        <!--                </svg>-->
+        <!--              </a>-->
+        <!--            </div>-->
+        <!--          </div>-->
+        <!--        </div>-->
```
Comment (akshay-kh) [inline src/pages/login.vue:114]: this also

**Review by akshay-kh — CHANGES_REQUESTED** [review]

File: src/components/Navbar.vue (hunk 1 above)
Comment (dipeshkemuri) [reply src/components/Navbar.vue]: Removed

File: src/components/Navbar.vue (hunk 2 above)
Comment (dipeshkemuri) [reply src/components/Navbar.vue]: Removed

File: src/pages/login.vue (hunk 3 above)
Comment (dipeshkemuri) [reply src/pages/login.vue]: Removed

**Review by akshay-kh — APPROVED** [review]

**Pull request merged**